ui/src/app/views/dataset-page.tsx

```diff
@@ -378,6 +380,33 @@ const DataSetPage = fp.flow(withCurrentWorkspace(), withUrlParams())(
       }
     }
 
+    isEllipsisActive(text) {
+      // For Data table with column title as active text, enable the tooltip if column's scroll
+      // width is greater than offsetWidth as that represents the textOverflow style has been
+      // applied and it has ellipsis at the end
+
+      const column = document.getElementsByClassName('p-column-title');
```

Comment:
Nit: I'd call this `columns` rather than `column` because it covers all columns.